cmd/move_config.go

```diff
@@ -66,12 +71,18 @@ func Move(dryRun bool) error {
 
 	log.Println("WARNING: Helm v3 configuration may be overwritten during this operation.")
 	log.Println()
-	doCleanup, err := utils.AskConfirmation("Move Config", "move the v2 configuration")
+	if skipConfirmation {
+		log.Println("Skipping confirmation before performing move.")
+		doCleanup = true
+		err = nil
+	} else {
+		doCleanup, err = utils.AskConfirmation("Move Config", "move the v2 configuration")
+	}
 	if err != nil {
```

Comment:
I think you can move this `err` check inside the above `else` clause. That way you don't need to declare `err` globally anymore.

Reply:
fixed